sklearn/svm/tests/test_svm.py

```diff
@@ -655,6 +655,46 @@ def test_linearsvc_crammer_singer():
     assert_array_almost_equal(dec_func, cs_clf.decision_function(iris.data))
 
 
+def test_linearsvc_fit_sampleweight():
+    # check correct result when sample_weight is 1
+    # check that SVR(kernel='linear') and LinearSVC() give
+    # comparable results
+
+    # Test basic routines using LinearSVC
+    n_samples = len(X)
+    unit_weight = np.ones(n_samples)
+    clf = svm.LinearSVC(random_state=0).fit(X, Y)
+    clf_unitweight = svm.LinearSVC(random_state=0).fit(X, Y,
+                                                  sample_weight=unit_weight)
```

Comment:
Please follow PEP8 indentation. An easy option is to do the `.fit` on a separate line.